test/Tizen.NUI.WindowSystem.Samples/Tizen.NUI.WindowSystem.Samples.cs

```diff
@@ -0,0 +1,322 @@
+﻿using System;
+using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
+using Tizen.NUI.Components;
+using Tizen.NUI.WindowSystem;
+
+namespace Tizen.NUI.WindowSystem.Samples
+{
+    class Program : NUIApplication
+    {
+        Shell.TizenShell tzShell;
+        Shell.QuickPanelClient qpClient;
+        Button BtnClient;        
+        TextLabel textClientVisible;
+        TextLabel textClientScrollable;
+        TextLabel textClientOrientation;
+
+        Shell.QuickPanelService qpService;
+        Shell.TizenRegion qpRegion;
+        Button BtnService;
+
+        protected override void OnCreate()
+        {
+            base.OnCreate();
+            Initialize();
+        }
+
+        void Initialize()
+        {
+            Window.Instance.KeyEvent += OnKeyEvent;
```

Comment:
Window.Instance will be deprecated.
Please use this 
       Window window = NUIApplication.GetDefaultWindow();

Reply:
I changed all Window.Instance in this code.